modules/terraform-zscc-gwlb-aws/main.tf

```diff
@@ -16,9 +17,23 @@ resource "aws_lb_target_group" "gwlb_target_group" {
     healthy_threshold   = var.healthy_threshold
     unhealthy_threshold = var.unhealthy_threshold
   }
+
+  target_failover {
```

Comment:
I had made this change locally but was getting a validation error.  Did you see this error before you upgraded the aws provider? this was added in v4.38 and I didn't check the release notes to find when it was fixed and thought our providers aws (v4.7.0) not that outdated!

Reply:
yeah this is why the version .46 was required